test/thermodynamics/equations_of_state/van_der_waals_state_equation_test.py

```diff
@@ -0,0 +1,73 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics.equations_of_state import van_der_waals_state_equation as waals_law
+
+# Description
+## Test example from https://studfile.net/preview/1772224/page:8/ example 2.24
+
+# Find pressure in state 1. For this example koefficients for van-der-vaals state equation
+
+# a = (V/nu)^2 * (p2 * T1 - p1 * T2) / (T2 - T1) = 0.188 [Pa*m^6]
+# b = V - nu * R * (T2 - T1) / (p2 - p1) = 4.225 * 10^(-5) [m^3]
+# V = 0.25 [liters]
+# nu = 1 [moles]
+# T = 300 [K]
+
+# Pressure should be 90 atm
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    t = Quantity(300 * units.kelvins)
+    v = Quantity(0.25 * units.liters)
+    nu = Quantity(1 * units.mole)
+    a = Quantity(0.188 * units.pascals * (units.meter ** 3 / units.mole) ** 2)
+    b = Quantity(4.225 * 1E-5 * units.meters ** 3 / units.moles)
+    Args = namedtuple("Args", ["t", "v", "nu", "a", "b"])
+    return Args(t=t, v=v, nu=nu, a=a, b=b)
+
+
+def test_basic_pressure(test_args):
+    result = waals_law.calculate_pressure(test_args.v, test_args.t, test_args.nu, test_args.a, test_args.b)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.pressure)
+    result_ = convert_to(result, units.atm).evalf(5)
+    assert result_ == approx(88.8, 0.01)
```

Comment:
It should be 90 atm, according to reference example.

Reply:
This is due to the transfer of the dimension from Pa to atm. 1 atm = 1.013 * 10^5 Pa. I rounded it up to 10^5 Pa and it immediately affected the result. I have now recalculated it without rounding, which I did last time.